src/Plugins/Primitives/Plane/Plane.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+** EPITECH PROJECT, 2024
+** Raytracer
+** File description:
+** Plane
+*/
+
+#include "Plane.hpp"
+
+Primitive::Plane::Plane()
+{
+    _position = Math::Point3D(0, 0, 0);
+    _axis = X;
+}
+
+Primitive::Plane::Plane(Primitive::Axis axis, double position)
+{
+    switch (axis) {
+        case X:
+            _position = Math::Point3D(position, 0, 0);
+            break;
+        case Y:
+            _position = Math::Point3D(0, position, 0);
+            break;
+        case Z:
+            _position = Math::Point3D(0, 0, position);
+            break;
+        default:
+            break;
+    }
+    _axis = axis;
+}
+
+Math::Point3D Primitive::Plane::hitPoint(const Raytracer::Ray &ray) const
+{
+    Math::Point3D rayOrigin = ray.origin();
+    Math::Vector3D rayDirection = ray.direction();
+
+
```

Comment:
Hello Mister Tranchet,

Why there is two space ?

Reply:
Hello Mister Fradet,

That's a very good question, I remove one just after.